src/routes/navigation/rootNavigator.tsx

```diff
@@ -1,14 +1,47 @@
-import React from "react";
-import {NavigationContainer} from '@react-navigation/native';
+import React, { useEffect, useState } from "react";
+import { NavigationContainer } from "@react-navigation/native";
+import { createNativeStackNavigator } from "@react-navigation/native-stack";
 import TabNavigator from "./TabNavigator";
+import SplashView from "../../views/SplashView";
+import AsyncStorage from "@react-native-async-storage/async-storage";
+
+const SplashScreenStack = createNativeStackNavigator();
 
 const RootNavigator = (): React.ReactElement => {
+  const [isAppFirstLaunch, setIsAppFirstLaunch] = useState(null);
```

Comment:
Maybe would be better to user some other name like `splashScreen` `setSplashScreen`